### PR TITLE
Customer Modeling Analysis

### PR DESCRIPTION
**Pull request by aryantiwari10**
## Related Issue 

- Info about the related issue 

- [x] LGM-SOC'21 Participant 
- [ ] DevIncept Participant
- [x] Contributor

Closes: #559

### Describe the add-ons or changes you've made

PROJECT TITLE : Customer Modeling Analysis

GOAL : To predict about the customers modeling using ANN .

WHAT I HAD DONE : In this project first I performed a exploratory data analysis on the Customer Modeling dataset which includes of data cleaning , data manipulation, data preprocessing , data visualization and after that I did training and testing , then I did the feature scaling and used ANN for the prediction. While in the EDA part I have included different plots for the different visualizations of our dataset .
During the prediction I kept the epochs as 200 and batch size as 256 using different activation function and the optimizers and got a accuracy score of 68 % which can be further increase by Hyper parameter Tuning.
Some plots which we used for visualizing the dataset are Histogram , Barplot , Boxplot, Heatmap , Scatter plot , Pairplot , Jointplot etc.

LIBRARIES:

PANDAS

NUMPY

MATPLOTLIB

SEABORN

SCIPY

SKLEARN

Keras

TensorFlow

Conclusion : We get to know why customers are withdrawing their account from bank.
The accuracy we got using ANN in 200 epochs is 68 % and the loss is 0.002 which can be further reduced durinh hyperparameter tuning .

-Aryan Tiwari

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Describe how it has been tested
Describe how have you verified the changes made

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots (Only for Front End and UI/UX Designers)

 Original           | Updated
 :--------------------: |:--------------------:
 Original Screenshot | Updated Screenshot |

### CODE REVIEW

**Review by prathimacode-hub — CHANGES_REQUESTED** [review]
Perfect notebook. But your README.md lacks information, it needs more detailing. @aryantiwari10

**Comment by aryantiwari10** [comment]
Mam I have added more details in README now you can check

**Review by prathimacode-hub — APPROVED** [review]
Crystal clear. Great job. Your notebooks are apt to the point. You nailed it. 💰🏆@aryantiwari10

**Comment by aryantiwari10** [comment]
Thanks a lot mam 😊
